### PR TITLE
Aid opinion

### PR DESCRIPTION
**Pull request by codemutatio**
**What does this PR do?**

1. Enables user  create a comment

2. allow user rate the aid course

### CODE REVIEW

File: server/controllers/aidComments.js [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    INTERNAL_SERVER_ERROR, CREATED, NOT_FOUND, getStatusText
+} from 'http-status-codes';
+
+const db = require('./promise').AidDb;
+
+const Comments = {
+    async create(req, res) {
+        try {
+            const query = {
+                _id: req.params.aidid,
```
Comment (Way2nnadi) [inline server/controllers/aidComments.js]: we use camelCasing in javascript. should be `req.params.aidId`. fix this in this file and the other files

File: server/controllers/aidComments.js [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+            };
+
+            const aid = await db.findOne(query);
+            if (!aid) {
```
Comment (Way2nnadi) [inline server/controllers/aidComments.js:18]: pad `if` statement with new lines

File: server/controllers/aidComments.js [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+                status: 'error',
+                message: getStatusText(INTERNAL_SERVER_ERROR),
+                error,
+            });
+        }
+    },
+
+    async getOne(req, res) {
+        const queryText = {
+            _id: req.params.aidid,
```
Comment (Way2nnadi) [inline server/controllers/aidComments.js]: should be `req.params.aidId`

File: server/controllers/aidComments.js [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+            const aid = await db.findOne(queryText);
+
+            if (!aid) {
+                return res.status(NOT_FOUND).send({
+                    status: 'error',
+                    message: getStatusText(NOT_FOUND),
+                });
+            }
+
+            const comment = await aid.comments.id(req.params.commentid);
```
Comment (Way2nnadi) [inline server/controllers/aidComments.js]: `req.params.commentId`

File: server/models/aid.js [hunk 5]
```diff
@@ -20,6 +42,12 @@ const aidSchema = new Schema(
             required: true,
             type: String,
         },
+        /* rating: {
+            max: 5,
+            min: 0,
+            required: true,
+            type: Number,
+        }, */
```
Comment (Way2nnadi) [inline server/models/aid.js]: delete this is not using

File: test/aid.js [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+    describe('GET /aids', () => {
+        it('it should GET all the Aids', async () => {
+            request(app)
+                .get('/api/v1/aids')
+                .set('Accept', 'application/json')
+                .expect('Content-Type', /json/)
+                .expect(200);
+        });
+    });
+});
```
Comment (Way2nnadi) [inline test/aid.js:18]: Does this test work?

Comment (codemutatio) [reply test/aid.js:18]: yes it does

**Review by enyeInc — APPROVED** [review]
LGTM

File: server/controllers/aidComments.js [hunk 7]
```diff
@@ -0,0 +1,138 @@
+/* eslint-disable camelcase */
+/* eslint-disable sort-keys */
+import {
+    INTERNAL_SERVER_ERROR, CREATED, NOT_FOUND, getStatusText
+} from 'http-status-codes';
+
+const db = require('./promise').AidDb;
+
+const Comments = {
+    async create(req, res) {
```
Comment (Merdoth) [inline server/controllers/aidComments.js:10]: What's does this entire function do?

Comment (codemutatio) [reply server/controllers/aidComments.js:10]: it handles the create, read and update for the comments

Comment (Merdoth) [reply server/controllers/aidComments.js:10]: Interesting

Comment (Merdoth) [reply server/controllers/aidComments.js:10]: This looks good to me

**Review by Merdoth — COMMENTED** [review]
I think this is ready to be merged

**Comment by enyeInc** [comment]
@codemutatio minor merge conflict to fix on `server/routes/index.js`

**Pull request merged**